resources/public/api/conf/3.0/application.raml

```diff
@@ -65,4 +65,12 @@ documentation:
             example: "AAZZ0123456789a"
         get: !include lossClaimType_retrieve.raml
         /change-type-of-claim:
-          post: !include lossClaimType_amend.raml
\ No newline at end of file
+          post: !include lossClaimType_amend.raml
+      /order:
+        /{taxYearClaimedFor}:
+        uriParameters:
+          taxYearClaimedFor:
+            description: 'The tax year to which the claim applies. The start year and end year must not span two tax years. The minimum tax year is 2019-20. No gaps are allowed, for example, 2022-24 is not valid.'
+            type: string
+            example: "2022-23"
+        put: !include lossClaimsOrder_amend.raml
```

Comment:
Extra indentation required for these lines since the `taxYearClaimedFor` is not currently appearing in the url path.